ext/net/02_tls.js

```diff
@@ -27,7 +27,15 @@
     return core.opAsync("op_tls_handshake", rid);
   }
 
+  function opTlsGetAlpnProtocol(rid) {
+    return core.opAsync("op_tls_get_alpn_protocol", rid);
+  }
+
   class TlsConn extends Conn {
+    getAgreedAlpnProtocol() {
+      return opTlsGetAlpnProtocol(this.rid);
+    }
+
```

Comment:
```suggestion
    getAgreedAlpnProtocol() {
      return opTlsGetAlpnProtocol(this.rid);
    }

```

I feel like this is something that could be returned from `handshake()`

Reply:
To clarify, I imagine `handshake()` returning a `Promise<TlsHandshakeInfo>` where `TlsHandshakeInfo` is the following:

```
interface TlsHandshakeInfo {
  alpnProtocol: string | null;
}
```

Reply:
Great idea. Done.